### ISSUE TITLE
Paypal: +2%

### ISSUE THREAD

**Issue by iliakan** [iliakan/javascript-nodejs#857]
Paypal реально грабительские комиссии дерёт. Чтобы оставить этот способ при оплате учебника и курсов, но показать людям, что он дорого стоит, предлагаю как-то обозначить, что стоимость будет +2% из-за комиссий. И, соответственно, генерироваться счета будут с увеличенной суммой.

Как полагаешь? Отобразишь в дизайне?

**Comment by iliakan** [comment]
Или, может, как-то иначе показать людям, что этот метод не так чтобы вери гут =)

**Comment by iliakan** [comment]
ping

**Comment by Bezart** [comment]
Да, правильно будет показать комиссию. Не думаю, что делать вывод за пользователя гут/не гут, это правильное решение, так как у каждого свои предпочтения. 

Я предлагаю сделать так: http://take.ms/YLng9
